src/scss/_utils.scss

```diff
@@ -70,3 +70,26 @@
     }
   }
 }
+
+@mixin media($breakpoint) {
+  @if $breakpoint == 'sm' {
```

Comment:
I feel like these should be variables so we don't have typos

Reply:
BTW this is 🔥I'm going to switch the way I'm doing the LinkList media query to this